### PR TITLE
Workaround for issue: 7124, ops-user Administrator group check

### PR DESCRIPTION
**Pull request by lcastellano**
Fixes # 7124, 7054

This PR temporary disables the check that is meant to guarantee that the ops-user does not belong to the Administrators group. The test is only skipped if an incorrect NotFound error is returned from the PSC. In the future we will validate the ops-user privileges independently from its Role or Group membership.

### CODE REVIEW

**Comment by lcastellano** [comment]
It seems to me to, but unfortunately that's what vCenter does. Before I added the check I mistakenly used admin for ops-user and ended up spending 30 minutes removing the permissions from all the different resources in the tree, to make my admin account working again. That was actually the reason why I added those checks. The correct approach is to perform a smarter validation before adding permissions to resources. But that is big change.

**Comment by mdubya66** [comment]
do we need to add tests for this?

**Comment by lcastellano** [comment]
For this to be exercised in an integration test we need Patrick's environment. An Active Directory local server must be added to the Identity Sources in PSC/VC. As for unit tests, I have used the Simulator, but the Simulator behaves correctly in this case. The actual PSC is the problem.

File: pkg/vsphere/rbac/rbac.go [hunk 1]
```diff
@@ -376,3 +392,19 @@ func (am *AuthzManager) getRoleName(resource *Resource) string {
...
+
+func isNotFoundError(ctx context.Context, err error) bool {
+	op := trace.FromContext(ctx, "isNotFoundError")
+
+	if soap.IsSoapFault(err) {
+		vimFault := soap.ToSoapFault(err).VimFault()
+		op.Debugf("Error type: %s", reflect.TypeOf(vimFault))
+
+		_, ok := soap.ToSoapFault(err).VimFault().(types.NotFound)
+		if ok == true {
```
Comment (hickeng) [inline pkg/vsphere/rbac/rbac.go]: just return `ok` ;)

**Pull request merged**